packages/editor/CodeMirror/createEditor.ts

```diff
@@ -261,6 +261,7 @@ const createEditor = (
 					}),
 					keyCommand('Tab', insertOrIncreaseIndent, true),
 					keyCommand('Shift-Tab', decreaseIndent, true),
+					keyCommand('Mod-Enter', insertBlankLine, true),
```

Comment:
Consider using `insertLineAfter` from `editorCommands/insertLineAfter.ts` here. `insertLineAfter` should handle list continuation and auto-indentation.

Reply:
That makes sense. Made the necessary changes.